db/seeds.rb

```diff
@@ -186,3 +186,11 @@
   r.location = boulder
   r.feed_format = "video"
 end.save!
+
+if Rails.env.development?
+  AdminUser.create!(
+    email: 'helios@mojotech.com',
+    password: 'mojojojo',
+    password_confirmation: 'mojojojo'
+  )
+end
```

Comment:
Should there be an else for this conditional to create an admin user for production?

Reply:
When rails boots up, it would automatically start with any file named development (if in development mode). It doesn't mean that the development stage isn't executed only once, and the difference between the environments has more to do with caching. Either way, the internet doesn't recommend any code of the format: 
if Rails.env.production?
  do_additional_work
end
so I'll try looking into something else...